src/python/turicreate/toolkits/image_similarity/image_similarity.py

```diff
@@ -63,6 +69,9 @@ def create(
     batch_size : int, optional
         If you are getting memory errors, try decreasing this value. If you
         have a powerful computer, increasing this value may improve performance.
+        
+    **kwargs : optional
+        Options for downstream methods like tc.nearest_neighbors.create().
```

Comment:
Please change `tc.nearest_neighbors.create()` to `:py:func:`turicreate.nearest_neighbors.create()`. That will cause the text, in our API docs, to be linked to the appropriate documentation page for that function.